chatbot_ner/settings.py

```diff
@@ -59,6 +83,9 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+if ELASTIC_APM_ENABLED:
```

Comment:
Can we please move lines 38-56 here and club everything, something like,
```python
_elastic_apm_enabled = (os.environ.get('ELASTIC_APM_ENABLED') or '').strip().lower()
ELASTIC_APM_ENABLED = ...
ELASTIC_APM_SERVER_URL = ...
if ELASTIC_APM_ENABLED:
    ELASTIC_APM = ...
    INSTALLED_APPS.append('elasticapm.contrib.django')
    MIDDLEWARE.append('elasticapm.contrib.django.middleware.TracingMiddleware')
```

Reply:
This is done